api/client-server/registration.yaml

```
@@ -294,11 +300,11 @@ paths:
       responses:
         200:
           description: |-
-            An SMS message has been sent to the specified phone number.
-            Note that this may be an SMS message containing the validation token or it may be informing
-            the user of an error.
+            A SMS message has been sent to the specified phone number. Note
```

Extreme nitpicking:
```suggestion
            An SMS message has been sent to the specified phone number. Note
```
Because https://www.grammar.com/a-vs-an-when-to-use/

---

I actually asked for it to be An -> A earlier in the PR. Didn't realize that was a rule of English :/

Sorry